public/assets/scss/_index.scss

```diff
@@ -451,7 +451,7 @@ h5 .btn-link {
     .holding {
         height: 50rem;
         background-image: url('../img/backgrounds/holding-mobile.png');
-        
```

Comment:
Extra spaces are automatically removed by my code editor. I think it is a good thing, so I am leaving it 😉